### DISCUSSION TITLE
Don't seem to be able to get Wormhole support?

### DISCUSSION THREAD

**Discussion by maxitimes** [wormhole-foundation/wormhole#348] [Developer Q&A] [answered]
Hi people, appreciate the help from here, 

I went to https://github.com/solana-labs/oyster/issues for support on user interface and being able to access my BTC which appears to have been turned into wBBTC and unusable, but no response in 2 days? 

How can i get my BTC back?

**Comment by leoluk** [comment, marked as answer]
This is something wormholebridge.com has to add (see https://github.com/certusone/wormhole/discussions/344)

**Reply by maxitimes** [reply]
wormholebridge.com support only offers a "Help" section and it links to this github repo!  

I am not getting an answer on https://github.com/solana-labs/oyster/issues and feel that $5k of stranded BTC is not something i am willing to let go in a hurry.  

Solana is such a great blockchain... but we need help / assistance if Solana is wanting us to send assets here

Can you please reach out to them and ask them to follow up. Thank you

**Reply by maxitimes** [reply]
![Can-Not-Send-BBTC-Back-to-eth](https://user-images.githubusercontent.com/13531192/131204548-f328abef-20ad-4b29-8fd2-c9a113f56832.PNG)

**Reply by leoluk** [reply]
> wormholebridge.com support only offers a "Help" section and it links to this github repo!

Which is unfortunate -- we can't provide support for a third party UI we haven't built. Pinged them about your issue!

**Reply by maxitimes** [reply]
Thanks for reaching out to them, your a legend.  I have been contacted.
